test/e2e/run_volume_test.go

```diff
@@ -1089,4 +1089,20 @@ RUN chmod 755 /test1 /test2 /test3`, ALPINE)
 		Expect(checkCtr.OutputToString()).To(ContainSubstring("foo"))
 		Expect(checkCtr.OutputToString()).To(ContainSubstring("bar"))
 	})
+
+	It("user-specified overlay supersedes image volume", func() {
+		err = podmanTest.RestoreArtifact(REDIS_IMAGE)
```

Comment:
is the restore actually needed?

Reply:
We need something with an image volume, Redis has an image volume, and the other tests in the file use it.

I can probably convert over to nginx which I think is cached?